src/runtime/module.cc

```diff
@@ -68,6 +68,9 @@ PackedFunc ModuleNode::GetFunction(const std::string& name, bool query_imports)
   if (query_imports) {
     for (Module& m : self->imports_) {
       pf = m.operator->()->GetFunction(name, query_imports);
+      if (pf != nullptr) {
+        return pf;
+      }
     }
   }
   return pf;
```

Comment:
Would it be worth (and correct) putting an assert like `ICHECK(pf != nullptr) << "unable to get the function pointer"` ?

Reply:
The expected behvaior of this function is to return nullptr for functions inavailable, so we don't want to put ICHECKhere